modulefiles/build.hera.intel

```diff
@@ -23,7 +25,12 @@ module load nemsiogfs/2.5.3
 module load landsfcutil/2.4.1
 module load wgrib2/2.0.8
 
-module use /scratch1/NCEPDEV/nems/emc.nemspara/soft/modulefiles
-module load hdf5_parallel/1.10.6
-module load netcdf_parallel/4.7.4
-module load esmf/8.0.0_ParallelNetCDF
+module load jasper/2.0.15
+module load zlib/1.2.11
+module load png/1.6.35
+
+module load hdf5/1.10.6
+module load netcdf/4.7.4
+#module load nccmp/1.8.7.0
+setenv NCCMP /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/v1.0.0-beta1/intel-18.0.5.274/impi-2018.0.4/nccmp/1.8.7.0/bin/nccmp
```

Comment:
Why?
If you load the nccmp module, you can declare `NCCMP=$(which nccmp)` in your script.

Reply:
@aerorahul Thanks for the suggestion.  Done at 16351ec.  Not sure how to replicate this for the gnu build, however.